### PR TITLE
Update voice registry system.md -- revised problem statement

### PR DESCRIPTION
**Pull request by jcstine**
Recommended: "discoverability" is the the broad problem to be solved by VRS.  Discoverability operationally defined as: finding, being found, and fully delivering the brand promise. For Maria's and Dan's review.

### CODE REVIEW

**Comment by Elizabeth-Robins** [comment]
The registry may _enable_ the brand to deliver, but the wording "fully delivering the brand promise" makes it sound like that is the responsibility of the VRS, where we have no control over the deliver of the brand information. Simply the access to the brand. Maybe I'm being too nitpicky on that wording but it seems like we just put a lot on the VRS's shoulders that is not within its scope. Read as **Discoverability operationally defined as: fully delivering the brand promise** should scare us.

**Comment by rogerkibbe** [comment]
I agree with Elizabeth about avoiding marketing hype like "fully delivering the brand promise". I can go with discoverability if and this is a big if, it's all about technical discoverability. The VRS is a service to voice assistant providers, not consumers. It provides
- A service that shows if a given name is associated with a voice app and how to reach that voice app (need to be careful on what how to reach means)
- An aid in disambiguation when there are multiple voice apps associated with a name
- A place for an entity to register their voice app and the associated name(s) for it

"Discoverability" in the voice world most often refers to consumers' ability to find and discover voice apps. While the VRS may aid in this by providing tooling to voice assistant platforms, it is not the solution to the discoverability challenge that the industry talks about (IMHO it's more of an advertising, promotion, awareness challenge that is not unique to voice and exists in both the Apple App Store and Google play store which have millions of apps, most of which are never discovered and thus never used. I will get off my soapbox now)

In short, I'm not a fan of using the term "discoverability" because it's a loaded term in the voice industry and while the VRS aids in the solution, it is not in and of itself, the solution

**Comment by jcstine** [comment]
Roger – and Elizabeth – thank you.

 

Great comments, and the concerns about the “brand promise” and the term discoverability are spot on.   So appreciate your work and support of this effort!  

 

I am all for another descriptive term, and I will send my marketing speak back to the dark cell where it belongs.  😊  

 

However:  the concern that led to the pull request and proposed text was this: I did not/do not see that we have – anywhere --  a cogently expressed value proposition(s) for VRS, especially in the form of WIFMs (what’s in it for me) for platform providers, the developers-managers-owners of independent voice assistants, and consumers.   The big questions of why – and for whom --  have been largely unanswered. 

 

Roger, you nail the enterprise value proposition below.    I see no reason why we should not adopt it verbatim for the master documents.  But might there be a consumer flip side to this conceptual disc – such as the value of disambiguation for the consumer/patient/client?  A confidence (dare I use the word “trust?”) that I will be connected to the destination of my desires?

 

Again, thank you – so grateful for what you are bringing to the Open Voice Network. 

 

Looking forward to our conversation tomorrow.

**Comment by maridob** [comment]
> I agree with Elizabeth about avoiding marketing hype like "fully delivering the brand promise". I can go with discoverability if and this is a big if, it's all about technical discoverability. The VRS is a service to voice assistant providers, not consumers. It provides
> 
> * A service that shows if a given name is associated with a voice app and how to reach that voice app (need to be careful on what how to reach means)
> * An aid in disambiguation when there are multiple voice apps associated with a name
> * A place for an entity to register their voice app and the associated name(s) for it
> 
> "Discoverability" in the voice world most often refers to consumers' ability to find and discover voice apps. While the VRS may aid in this by providing tooling to voice assistant platforms, it is not the solution to the discoverability challenge that the industry talks about (IMHO it's more of an advertising, promotion, awareness challenge that is not unique to voice and exists in both the Apple App Store and Google play store which have millions of apps, most of which are never discovered and thus never used. I will get off my soapbox now)
> 
> In short, I'm not a fan of using the term "discoverability" because it's a loaded term in the voice industry and while the VRS aids in the solution, it is not in and of itself, the solution

+1 on @Elizabeth-Robins and @rogerkibbe POV.

**Comment by Elizabeth-Robins** [comment]
Jon, I totally agree with the value proposition concern as I have a similar question. For me, this leads to the larger issue of what happens after "discovery"? I'd like to understand the proposals associated with "delivery". If a Google device gets a request for a voice app and the VRS discovers it is in the Amazon marketplace, does that Skill get delivered to Google Assistant? I'm still getting up to speed on all this but it seems without complete interoperability (cooperation) between the voice services, the VRS is difficult to justify. Perhaps we can discuss briefly this morning.

**Comment by maridob** [comment]
Based on the meeting, re-write the proposal based on feedback from @rogerkibbe and @Elizabeth-Robins. Do not merge until re-write happen.

File: components/voice_registry_system.md [hunk 1]
```diff
@@ -10,11 +10,25 @@ VRS's goal is to provide and promote voice openness and transparency.
...
+The task of the Open Voice Network Voice Registry System (VRS) Work Group is to develop standards-based solutions for how _verbalized, explicitly-requested conversational agent destinations_ will,by another conversational agent, **be uniquely identified and connected.**
+
+This issue anticipates the continued growth of voice assistant systems that are independent of major platform technologies, and the emergence of a hybrid voice assistant ecosystem consisting of both general-purpose platform-based conversational agents and independent systems, most often managed and operated by enterprises. 
+
+The VRS will provide value to voice assistant system providers and operators, and to consumers. It will provide
```
Comment (maridob) [inline components/voice_registry_system.md:17]: who are the operators? Is this define somewhere?

**Review by maridob — COMMENTED** [review]
Overall, looks good. We just need to clarify the word operator

File: components/voice_registry_system.md (hunk 1 above)
Comment (maridob) [reply components/voice_registry_system.md:17]: The VRS will provide value to voice assistant system providers, organizations, content creators, and consumers. 

>> Discussed 06.10

**Review by maridob — APPROVED** [review]

**Comment by pmotch** [comment]
Curious what the latest is on this PR?
